distro/run/core/src/main/java/org/camunda/bpm/run/CamundaBpmRunConfiguration.java

```diff
@@ -59,6 +61,11 @@ public AdministratorAuthorizationPlugin administratorAuthorizationPlugin() {
   @Bean
   public ProcessEngineConfigurationImpl processEngineConfigurationImpl(List<ProcessEnginePlugin> processEnginePlugins) {
     final SpringProcessEngineConfiguration configuration = new CamundaBpmRunProcessEngineConfiguration();
+
+    // register process engine plugins defined in yaml
+    Map<String, Map<String, Object>> yamlPluginsInfo = camundaBpmRunProperties.getProcessEnginePlugins();
```

Comment:
I think this part does not work with plugins that don't have any properties.
We should probably also add a test case that covers plugins without properties.

Reply:
You're right. I overlooked that use-case. The YAML-to-Map mapping won't allow us to provide plugin configurations without parameters. I will need to refactor the `yamlPluginsInfo` data structure. 

I will provide a `List<CamundaBpmRunProcessEnginePluginProperty>` instead. The `CamundaBpmRunProcessEnginePluginProperty` class will have:
* a `String pluginClass` property
* a `Map pluginParameters` property

This approach will allow users to configure the same plugin twice. However, I think this is acceptable since it is a variation of a "plugin override" scenario. The last plugin configuration will be applied.